android/src/main/java/com/reactnativecommunity/asyncstorage/next/StorageModule.kt

```diff
@@ -16,7 +16,7 @@ import kotlinx.coroutines.SupervisorJob
 import kotlinx.coroutines.asExecutor
 import kotlinx.coroutines.launch
 
-class StorageModule(reactContext: ReactContext) : ReactContextBaseJavaModule(), CoroutineScope {
+class StorageModule(reactContext: ReactApplicationContext) : ReactContextBaseJavaModule(reactContext), CoroutineScope {
```

Comment:
Interesting - did `ReactContextBaseJavaModule` always accepted context or was it added in new RN version?